### ISSUE TITLE
Port SGMCMCJax to blackjax

### ISSUE THREAD

**Issue by rlouf** [blackjax-devs/blackjax#289]
As discussed with the author of SGMCMCJax @jeremiecoullon we will be porting SGMCMCJax to blackjax.

## Gradient estimation

- [x] Standard gradient estimator
- [ ] Control variate gradient estimator
- [ ] SVRG gradient estimator

# Integrators

- [x] SgLD
- [ ] pSgLD
- [ ] SgldAdam
- [x] SgHMC
- [ ] baoab
- [ ] sgnht
- [ ] badodab
